src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,16 +27,39 @@
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
+    public static final String MESSAGE_NO_NEXT_SESSION_FOUND = "Don't worry, no next session for you this week!";
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
-
+    private final Session timeNowSession;
+    private final ArrayList<Session> toSortList;
+    private final HashMap<Session, Person> sessionPersonHashMap;
     private ObservableList<TimeSlot> timeSlots;
     private boolean isDayView;
     private boolean isFullView;
 
+    /**
+     * Class implementing Consumer acting as a helper to add Sessions later
+     * than current time in the week to the list to be sorted.
+     */
+    private Consumer<Person> toSortListAdder = new Consumer<Person>() {
+        @Override
+        public void accept(Person person) {
+            if (person.getSessionList().sessionList.isEmpty()) {
```

Comment:
like joanna mentioned earlier, this if statement is pointless since if it was empty your for loop would essentially just return and not run.

Reply:
sry abt thiss